pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -0,0 +1,662 @@
+			}},
+			Strategy: appsv1.DeploymentStrategy{
+				RollingUpdate: &appsv1.RollingUpdateDeployment{
+					MaxUnavailable: &maxUnavailable,
+					MaxSurge:       &maxSurge,
+				},
+				Type: appsv1.RollingUpdateDeploymentStrategyType,
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						v1beta1constants.GardenRole: "controlplane",
+						"garden.sapcloud.io/role":   "controlplane",
+						"app":                       DeploymentName,
+						"networking.gardener.cloud/to-shoot-networks":        v1beta1constants.LabelNetworkPolicyAllowed,
+						v1beta1constants.LabelNetworkPolicyToDNS:             v1beta1constants.LabelNetworkPolicyAllowed,
+						v1beta1constants.LabelNetworkPolicyToPrivateNetworks: v1beta1constants.LabelNetworkPolicyAllowed,
+						v1beta1constants.LabelNetworkPolicyFromPrometheus:    v1beta1constants.LabelNetworkPolicyAllowed,
+					},
+					Annotations: map[string]string{
+						"scheduler.alpha.kubernetes.io/critical-pod": "",
+						"checksum/secret-" + tlsAuthSecret.Name:      v.secrets.TLSAuth.Checksum,
+						"checksum/secret-" + dhSecret.Name:           v.secrets.DH.Checksum,
+					},
+				},
+				Spec: corev1.PodSpec{
+					PriorityClassName: "system-cluster-critical",
+					DNSPolicy:         corev1.DNSDefault, // make sure to not use the coredns for DNS resolution.
+					Containers: []corev1.Container{
+						{
+							Name:            DeploymentName,
+							Image:           v.imageVPNSeedServer,
+							ImagePullPolicy: corev1.PullIfNotPresent,
+							Ports: []corev1.ContainerPort{
+								{
+									Name:          "tcp-tunnel",
+									ContainerPort: 1194,
+									Protocol:      corev1.ProtocolTCP,
+								},
+							},
+							SecurityContext: &corev1.SecurityContext{
+								Capabilities: &corev1.Capabilities{
+									Add: []corev1.Capability{
+										"NET_ADMIN",
+									},
+								},
+								Privileged: pointer.BoolPtr(true),
+							},
+							Env: []corev1.EnvVar{
+								{
+									Name:  "SERVICE_NETWORK",
+									Value: v.serviceNetwork,
+								},
+								{
+									Name:  "POD_NETWORK",
+									Value: v.podNetwork,
+								},
+							},
+							Resources: corev1.ResourceRequirements{
+								Requests: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse("100m"),
+									corev1.ResourceMemory: resource.MustParse("100Mi"),
+								},
+								Limits: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse("1"),
+									corev1.ResourceMemory: resource.MustParse("1Gi"),
+								},
+							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      DeploymentName,
+									MountPath: "/srv/secrets/vpn-server",
+								},
+								{
+									Name:      "vpn-seed-server-tlsauth",
+									MountPath: "/srv/secrets/tlsauth",
+								},
+								{
+									Name:      "vpn-seed-server-dh",
+									MountPath: "/srv/secrets/dh",
```

Comment:
define constants for the mount paths

Reply:
The mount paths are only used once in the volumeMounts section. What benefit does a constant have in this case? From my perspective it reduces the readability because the path is not directly visible.

Reply:
OK, I understand. Wouldn't it be a better practice to then make the paths configurable in the VPN component (e.g., via env variables) instead of hard-coding them there and "duplicating" them here? Even if it's unlikely that they change, it's still potential for failure.